### ISSUE TITLE
S3 URL returned by GraphQL query yields 404 for ds000248

### ISSUE THREAD

**Issue by hoechenberger** [OpenNeuroOrg/openneuro#2521]
**Describe the bug**
An S3 URL returned by the GraphQL API yields a 404 error.

**To Reproduce**

The following GraphQL query:

```graphql
query {
  dataset(id: "ds000248") {
    latestSnapshot {
      id
      files(
        prefix: "sub-01/anat/sub-01_T1w.nii.gz"
      ) {
        urls
      }
    }
  }
}
```
yields:
```graphql
{
  "data": {
    "dataset": {
      "latestSnapshot": {
        "id": "ds000248:1.2.4",
        "files": [
          {
            "urls": [
              "https://s3.amazonaws.com/openneuro.org/ds000248/sub-01/anat/sub-01_T1w.nii.gz?versionId%3D8uTXIQ10Blcp2GeAVJJCHL5PimkSaQZL"
            ]
          }
        ]
      }
    }
  },
...
```
However, accessing the produced URL,
```
https://s3.amazonaws.com/openneuro.org/ds000248/sub-01/anat/sub-01_T1w.nii.gz?versionId%3D8uTXIQ10Blcp2GeAVJJCHL5PimkSaQZL
```
yields a 404 error.

**Expected behavior**
The URL should point to the requested file.

**Additional context**
Downloading through the OpenNeuro website works.

**Comment by nellh** [comment]
@hoechenberger Thanks for this bug report, a fix for this is deployed in OpenNeuro 4.3.1.

**Comment by hoechenberger** [comment]
Thank you, @nellh!
